### DISCUSSION TITLE
Changing Scroll Speed/Amount

### DISCUSSION THREAD

**Discussion by jzhang172** [idiotWu/smooth-scrollbar#529] [Q&A] [answered]
I am using Angular and not sure how to implement:

https://stackblitz.com/edit/angular-ivy-a7uhw1?file=src%2Fapp%2Fapp.component.ts,src%2Fapp%2Fapp.component.html,node_modules%2Fsmooth-scrollbar%2Fplugin.d.ts

**Comment by sadeghbarati** [comment]
https://codepen.io/idiotWu/pen/VwbborM

You can put your class plugin outside of init

Also you have to extends ScrollbarPlugin

**Reply by jzhang172** [reply]
Thank you!

**Comment by dolphin-wood** [comment, marked as answer]
1. Your plugin needs to extend the base `ScrollbarPlugin`:

```ts
class ScaleSpeedPlugin extends ScrollbarPlugin {
  // ...
}
```

2. Fix the class name in `Scrollbar.use`:

```ts
Scrollbar.use(ScaleSpeedPlugin);
```

**Reply by jzhang172** [reply]
Thank you!
